spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigurationChangeDetector.java

```diff
@@ -164,7 +163,7 @@ else if (BootstrapPropertySource.class.isInstance(source)) {
 	}
 
 	private <E> LinkedList<E> toLinkedList(Iterable<E> it) {
-		LinkedList<E> list = new LinkedList<E>();
```

Comment:
diamond operator exists since java-7, so I removed the not needed extra type parameter